graphql-dxm-provider/src/main/resources/tools/graphql-playground.jsp

```diff
@@ -0,0 +1,57 @@
+<%@ page contentType="text/html;charset=UTF-8" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+
+<c:set var="modulePath" value="${pageContext.request.contextPath}/modules/graphql-dxm-provider"/>
+<c:set var="graphQlEndpoint" value="${pageContext.request.contextPath}/modules/graphql"/>
+
+<!doctype html>
+<html>
+    <head>
+        <meta charset="UTF-8"/>
+        <title>GraphQL Playground</title>
+        <link rel="stylesheet" href="${modulePath}/css/graphql-playground.css" type="text/css"/>
+    </head>
+    <body>
+        <div id="graphql-playground">
+            <p class="loading">Loading <span class="title">GraphQL Playground</span></p>
+        </div>
+        <script src="${modulePath}/javascript/graphql-playground-1.7.23.js"></script>
+        <script>
+            const customTheme = {
+                // property: '',
+                // comment: '',
+                // punctuation: '',
+                //keyword: '',
+                // def: '',
+                // qualifier: '',
+                // attribute: '',
+                // number: '',
+                // string: '',
+                // builtin: '',
+                // string2: '',
+                // variable: '',
+                // meta: '',
+                // atom: '',
+                // ws: '',
+                // selection: '',
+                // cursorColor: '',
+                // editorBackground: '',
+                // resultBackground: '',
+                // leftDrawerBackground: '',
+                // rightDrawerBackground: ''
+            };
+
+            window.addEventListener('load', function() {
+                GraphQLPlayground.init(document.getElementById('graphql-playground'), {
+                    endpoint: '${graphQlEndpoint}',
+                    settings: {
+                        'request.credentials': 'same-origin'
+                    },
+                    shareEnabled: true,
```

Comment:
This one enables integration with https://www.graphqlbin.com/ to share a workspace. Don't know whether or not we should enable that. Should double check with @fpral

Reply:
> I don't recall exactly why I did not use 1.8.x branch in the end (stability?). But a 1.7.26 has been released in between, thus we should use this one over the 1.7.23 one.

I couldn't use 1.7.26 version tried to make it work but It doesn't load constantly, here is a screenshot
<img width="1436" alt="Capture d’écran 2020-09-01 à 09 07 55" src="https://user-images.githubusercontent.com/12768472/91813348-e0aa5280-ec32-11ea-8616-da47557ddcd9.png">